code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -505,6 +505,30 @@
 	icon_opened = "electricalcrate_open"
 	icon_closed = "electricalcrate"
 
+/obj/structure/closet/crate/mail
+	name = "mail crate"
+	desc = "A plastic crate for mail delivery."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mailsealed"
+	icon_opened = "mailopen"
+	icon_closed = "mailsealed"
+	material_drop = /obj/item/stack/sheet/plastic
+	material_drop_amount = 4
+	var/list/possible_contents = list(/obj/item/envelope/security,
+										/obj/item/envelope/science,
+										/obj/item/envelope/supply,
+										/obj/item/envelope/medical,
+										/obj/item/envelope/engineering,
+										/obj/item/envelope/service,
+										/obj/item/envelope/command,
+										/obj/item/envelope/misc)
+
+/obj/structure/closet/crate/mail/populate_contents()
+	. = ..()
+	for(var/i in 1 to rand(5, 10))
```

Comment:
I wanna see how cargo will act having trying to deliver 10 mails :D

Reply:
I reckon they'll cope, if not the amount can be reduced in a later PR.

Reply:
I honestly think it's too little to do in 15 minutes haha